### ISSUE TITLE
Issue with commonjs@17

### ISSUE THREAD

**Issue by frederikhors** [PepsRyuu/nollup#161]
We just fixed the issue with `nollup` and `commonjs@16`; now we have one with `commonjs@17`:

```
Error: Insufficient Rollup version: "@rollup/plugin-commonjs" requires at least rollup@2.30 but found rollup@2.0.
  at validateRollupVersion (C:\svelte-template-hot\node_modules\@rollup\plugin-commonjs\dist\index.js:591:11)
  at Object.buildStart (C:\svelte-template-hot\node_modules\@rollup\plugin-commonjs\dist\index.js:1504:7)
  at _callAsyncHook (C:\svelte-template-hot\node_modules\nollup\lib\impl\PluginLifecycle.js:12:31)    at callAsyncParallelHook (C:\svelte-template-hot\node_modules\nollup\lib\impl\PluginLifecycle.js:68:26)
  at Object.buildStart (C:\svelte-template-hot\node_modules\nollup\lib\impl\PluginLifecycle.js:116:19)
  at Object.compile (C:\svelte-template-hot\node_modules\nollup\lib\impl\NollupCompiler.js:206:41)    at generateImpl (C:\svelte-template-hot\node_modules\nollup\lib\index.js:13:42)
  at C:\svelte-template-hot\node_modules\nollup\lib\index.js:38:21
  at new Promise (<anonymous>)
  at Object.generate (C:\svelte-template-hot\node_modules\nollup\lib\index.js:34:20)
```

Reproduction: https://github.com/frederikhors/svelte-template-hot/tree/update-to-commonjs-17

**Comment by flayks** [comment]
Second that, I was about to post an issue about it 🙈

**Comment by PepsRyuu** [comment]
Does it work by changing line 12 here: https://github.com/PepsRyuu/nollup/blob/master/lib/impl/PluginContext.js

Try change that to "2.30".

**Comment by frederikhors** [comment]
> Try change that to "2.30". I'm assuming that's all it is and that there's nothing else to look into.

IT WORKS!

**Comment by PepsRyuu** [comment]
Fixed in ```0.14.1```.
